ibm_vpd_utils.cpp

```diff
@@ -1133,5 +1159,67 @@ void getBackupRecordKeyword(std::string& record, std::string& keyword)
         }
     }
 }
+
+void clearVpdOnRemoval(const std::string& objPath,
```

Comment:
just a thought here,
as part of clearing the VPD when the FRU is removed, instead of emptying all keywords,
why don't we reset the dbus object state back to how it was when we prime the inventory FRUs? (with just its Location code and Present=false set)
In this way we don't need to track vpd specific interfaces and its properties.

Reply:
There are persisted data which is causing the confusion here. Hence the requirement is to get rid of VPD.

Reply:
we can get rid of VPD by priming the inventory also right?

Reply:
> we can get rid of VPD by priming the inventory also right?

How? You will create an empty map of object path and all the interfaces that it hosts or by calling the existing function to prime the inventory?